x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/results/AnomalyRecord.java

```diff
@@ -253,6 +261,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeList(influences);
         }
         out.writeOptionalWriteable(geoResults);
+        out.writeOptionalWriteable(anomalyScoreExplanation);
```

Comment:
Extra protection is need in a mixed version cluster as pre 8.6 nodes do not know about `anomalyScoreExplanation`

```
if (in.getVersion().onOrAfter(Version.V_8_6_0)) {
   out.writeOptionalWriteable(anomalyScoreExplanation);
}
```

Similarly for the ctor where the field is read

Reply:
Good point 👍 Fixed in a0e3327.